### PR TITLE
chore: update reflect-metadata to 0.1.1

### PR DESCRIPTION
**Pull request by tamascsaba**
Update reflect-metadata to 0.1.1

### CODE REVIEW

**Comment by vicb** [comment]
see https://github.com/angular/angular/blob/master/npm-shrinkwrap.readme.md

**Comment by tamascsaba** [comment]
Ohh, thanks, I will update the pull request.

**Comment by tamascsaba** [comment]
Updated :)

File: npm-shrinkwrap.json [hunk 1]
```diff
@@ -14591,8 +14591,8 @@
     },
     "reflect-metadata": {
       "version": "0.1.0",
```
Comment (alfonso-presa) [inline npm-shrinkwrap.json]: You have forgotten this one :-).

Comment (tamascsaba) [inline npm-shrinkwrap.json]: Fixed, thanks :)

**Comment by tamascsaba** [comment]
This PR solve: https://github.com/angular-class/angular2-webpack-starter/issues/45

**Pull request closed without merging**